frontend/packages/console-app/src/components/lightspeed/Lightspeed.scss

```diff
@@ -0,0 +1,101 @@
+@import '~@patternfly/patternfly/sass-utilities/all';
+@import '../../../../../public/style/vars';
```

Comment:
is this styling from the lightspeed plugin ?

Reply:
It is duplicated and modified styling from the Lightspeed plugin as:
1. Mark requested some styling differences 
2. I added support for dark mode (which the plugin does not currently support) but I expect Andy will adopt in the plugin